rewrite-java/src/main/java/org/openrewrite/java/UpdateSourcePositions.java

```diff
@@ -66,14 +66,15 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
             }
 
             @Override
-            protected void visitModifier(J.Modifier modifier, PrintOutputCapture<ExecutionContext> p) {
+            public J visitModifier(J.Modifier modifier, PrintOutputCapture<ExecutionContext> p) {
```

Comment:
I suspect that this method can now probably be entirely removed, as it will be covered by the `visit()` method now.